internal/packages/packageloader/transformfiles_funcs.go

```diff
@@ -0,0 +1,258 @@
+
+	// Until
+	"until":     {},
+	"untilStep": {},
+
+	// Arithmetic
+	"add1":    {},
+	"add":     {},
+	"sub":     {},
+	"div":     {},
+	"mod":     {},
+	"mul":     {},
+	"randInt": {},
+	"add1f":   {},
+	"addf":    {},
+	"subf":    {},
+	"divf":    {},
+	"mulf":    {},
+	"biggest": {},
+	"max":     {},
+	"min":     {},
+	"maxf":    {},
+	"minf":    {},
+	"ceil":    {},
+	"floor":   {},
+	"round":   {},
+
+	// Join
+	"join":      {},
+	"sortAlpha": {},
+
+	// Defaults
+	"default":          {},
+	"empty":            {},
+	"coalesce":         {},
+	"all":              {},
+	"any":              {},
+	"compact":          {},
+	"mustCompact":      {},
+	"fromJson":         {},
+	"toJson":           {},
+	"toPrettyJson":     {},
+	"toRawJson":        {},
+	"mustFromJson":     {},
+	"mustToJson":       {},
+	"mustToPrettyJson": {},
+	"mustToRawJson":    {},
+	"ternary":          {},
+	"deepCopy":         {},
+	"mustDeepCopy":     {},
+
+	// Reflection
+	"typeOf":     {},
+	"typeIs":     {},
+	"typeIsLike": {},
+	"kindOf":     {},
+	"kindIs":     {},
+	"deepEqual":  {},
+
+	// Paths
+	"base":  {},
+	"dir":   {},
+	"clean": {},
+	"ext":   {},
+	"isAbs": {},
+
+	// Encoding
+	"b64enc": {},
+	"b64dec": {},
+	"b32enc": {},
+	"b32dec": {},
+
+	// Data Structures
+	"tuple":              {},
+	"list":               {},
+	"dict":               {},
+	"get":                {},
+	"set":                {},
+	"unset":              {},
+	"hasKey":             {},
+	"pluck":              {},
+	"keys":               {},
+	"pick":               {},
+	"omit":               {},
+	"merge":              {},
+	"mergeOverwrite":     {},
+	"mustMerge":          {},
+	"mustMergeOverwrite": {},
+	"values":             {},
+	"append":             {},
+	"mustAppend":         {},
+	"push":               {},
+	"mustPush":           {},
+	"prepend":            {},
+	"mustPrepend":        {},
+	"first":              {},
+	"mustFirst":          {},
+	"rest":               {},
+	"mustRest":           {},
+	"last":               {},
+	"mustLast":           {},
+	"initial":            {},
+	"mustInitial":        {},
+	"reverse":            {},
+	"mustReverse":        {},
+	"uniq":               {},
+	"mustUniq":           {},
+	"without":            {},
+	"mustWithout":        {},
+	"has":                {},
+	"mustHas":            {},
+	"slice":              {},
+	"mustSlice":          {},
+	"concat":             {},
+	"dig":                {},
+	"chunk":              {},
+	"mustChunk":          {},
+
+	// Crypto
+	"bcrypt":                   {},
+	"htpasswd":                 {},
+	"genPrivateKey":            {},
+	"derivePassword":           {},
+	"buildCustomCert":          {},
+	"genCA":                    {},
+	"genCAWithKey":             {},
+	"genSelfSignedCert":        {},
+	"genSelfSignedCertWithKey": {},
+	"genSignedCert":            {},
+	"genSignedCertWithKey":     {},
+	"encryptAES":               {},
+	"decryptAES":               {},
+	"randBytes":                {},
```

Comment:
A lot of these functions are also producing randomized outputs.
Creating and working with certificates within our template process sounds like a really bad idea, I would like to leave those out as well.

Reply:
:+1:

Reply:
see [here](https://github.com/package-operator/package-operator/pull/172#issuecomment-1431244223)